perllib/FixMyStreet/App/Controller/Around.pm

```diff
@@ -253,6 +253,7 @@ sub map_features : Private {
 
     $c->forward( '/reports/stash_report_filter_status' );
     $c->forward( '/reports/stash_report_sort', [ 'created-desc' ]);
+    $c->stash->{show_old_reports} = $c->get_param('show_old_reports') && $c->get_param('show_old_reports') eq '1' ? 'on' : 'off';
```

Comment:
Can this not be `$c->get_param('show_old_reports') ? 'on' : 'off';`, if it's either set or not set?

Reply:
And then perhaps reduce it to only `$c->get_param('show_old_reports')` and then you can do `IF show_old_reports` rather than have to compare to a string.